### PR TITLE
DOC-4228: New pages for cdm

### PR DESCRIPTION
**Pull request by beajohnson**

### CODE REVIEW

**Review by msmygit — APPROVED** [review]
lgtm 👍🏼

File: modules/ROOT/pages/cdm-steps.adoc [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+[IMPORTANT]
+====
+The {cstar-data-migrator} validation job will never delete records from Target.
+The job only adds or updates data on Target.
+====
+
+[[cdm-guardrail-checks]]
+== Perform large-field guardrail violation checks
+
+Use {cstar-data-migrator} to identify large fields from a table that may break your cluster guardrails.
```
Comment (msmygit) [inline modules/ROOT/pages/cdm-steps.adoc:89]:
```suggestion
Use {cstar-data-migrator} to identify large fields from a table that may break your https://docs.datastax.com/en/astra-db-serverless/cql/cassandra-guardrails.html[cluster guardrails].
```

**Review by jeromatron — APPROVED** [review]
I think this will be great to have.  We might just want to double check with Faizal and Pravin and the CDM team are happy with these details.

**Review by KimberlyFields — CHANGES_REQUESTED** [review]

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -24,8 +24,12 @@
 * xref:change-read-routing.adoc[]
 //phase 5
 * xref:connect-clients-to-target.adoc[]
+* xref:cassandra-data-migrator.adoc[]
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/nav.adoc]: Canceling the ticket DOC-4235 as I've added the nav to this ticket.

File: modules/ROOT/pages/cassandra-data-migrator.adoc [hunk 3]
```diff
@@ -1,30 +1,14 @@
-= {cstar-data-migrator}
+= Introduction to {cstar-data-migrator}
```
Comment (KimberlyFields) [inline modules/ROOT/pages/cassandra-data-migrator.adoc:1]: Remove "Introduction to". This page ends up including everything about CDM, so it's not just an intro.

File: modules/ROOT/pages/cassandra-data-migrator.adoc [hunk 4]
```diff
@@ -53,678 +37,21 @@ cd cassandra-data-migrator
 mvn clean package
 ----
 
-The fat jar (`cassandra-data-migrator-x.y.z.jar`) file should be present now in the `target` folder.
+The fat jar, or`cassandra-data-migrator-x.y.z.jar`, file should be present now in the `target` folder.
```
Comment (KimberlyFields) [inline modules/ROOT/pages/cassandra-data-migrator.adoc:40]:
```suggestion
The fat jar file, `cassandra-data-migrator-x.y.z.jar`, should be present now in the `target` folder.
```

**Review by KimberlyFields — CHANGES_REQUESTED** [review]

File: modules/ROOT/nav.adoc [hunk 5]
```diff
@@ -1,34 +1,45 @@
...
 * xref:glossary.adoc[]
+
 * xref:contributions.adoc[]
-* xref:release-notes.adoc[]
\ No newline at end of file
+
+* xref:release-notes.adoc[]
+
+* {cstar-data-migrator}
+** xref:cassandra-data-migrator.adoc[Overview]
```
[the remark this answers is not in the document]
Comment (beajohnson) [reply modules/ROOT/nav.adoc]: I've tried making partials with this nav full of xrefs and keep getting errors. Unfortunately, the links have to remain xrefs.

File: modules/ROOT/pages/cdm-steps.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
+= {cstar-data-migrator}
```
Comment (KimberlyFields) [inline modules/ROOT/pages/cdm-steps.adoc:1]:
```suggestion
= Migrate data using the {cstar-data-migrator}
```

File: modules/ROOT/nav.adoc
Comment (KimberlyFields) [inline modules/ROOT/nav.adoc:1]: Do you still need this nav file in this PR if you are going to create a separate PR for the overall nav updates for migration?

**Review by KimberlyFields — APPROVED** [review]
I made 2 comments, but I'm approving this PR.

File: modules/ROOT/nav.adoc
Comment (beajohnson) [reply modules/ROOT/nav.adoc:1]: I like this nav best and it has the most changes.

**Pull request merged**